src/codeu/chat/client/commandline/Chat.java

```diff
@@ -24,6 +24,9 @@
 import codeu.chat.client.core.MessageContext;
 import codeu.chat.client.core.UserContext;
 
+import codeu.chat.common.ServerInfo;
```

Comment:
nit: imports should be ordered alphabetically (https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing)